mysql-test/suite/innodb/t/bulk_copy_alter.combinations

```diff
@@ -0,0 +1,5 @@
+[bulk_alter_copy]
+--innodb_alter_copy_bulk=ON
+
+[non_bulk_alter_copy]
+--innodb_alter_copy_bulk=OFF
```

Comment:
I think that we should use shorter names if we rename the test (and the `.combinations` file) to more self-descriptive `innodb_alter_copy_bulk` or `alter_copy_bulk`.
```
[ON]
--innodb_alter_copy_bulk=ON

[OFF]
--innodb_alter_copy_bulk=OFF
```
Then the test log will display `innodb.alter_copy_bulk,ON` which can intuitively be interpreted as a test of `innodb_alter_copy_bulk=ON`.